packages/stanislav-sequence/src/Exercise2/CountSubsequenceJS.ts

```diff
@@ -0,0 +1,10 @@
+//name: CountSubsequenceJS 
+// input: string sequence
+// input: string subsequence
+// output: int count 
+//language: javascript
+
+export function complementWidget(sequence: any, subsequence: any): any {
```

Comment:
you can't use 'any' so much
additionallym, these are definitely the strings, why using any?